sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -153,7 +153,11 @@ SUBROUTINE CALRH_NAM(P1,T1,Q1,RH)
                 Q1(I,J) = RH(I,J)*QC
               ENDIF
               IF (RH(I,J) < RHmin) THEN  !use smaller RH limit for stratosphere
-                RH(I,J) = RHmin
+                IF (P1(I, J) >= 3e2) THEN
+                  RH(I,J) = RHmin
+                ELSE IF (RH(I, J) < (RHmin / 10)) THEN
```

Comment:
@DWesl I suggest changing "RHmin / 10" into "RHmin / 10.".  Also please add a change log entry with your name next to line 119 as "!> 2025-04-16 | ..."